GDASApp WORKFLOW_TESTS=ON build breaks with g-w PR #3745

The GDASApp build with `WORKFLOW_TESTS="ON"` fails with the following error when built inside g-w `develop` at and after [`3b1cf51`](https://github.com/NOAA-EMC/global-workflow/commit/3b1cf516b2cd3d14b229cade3d39c1468e687d92):

```
FileNotFoundError: [Errno 2] No such file or directory: '/lfs/h2/emc/da/noscrub/russ.treadon/git/global-workflow/test/sorc/gdas.cd/bundle/../../../dev/ci/cases/pr/C96C48_hybatmaerosnowDA.yaml'
CMake Error at gdas/test/gw-ci/CMakeLists.txt:231 (string):
```

g-w `develop` at [`3b1cf51`](https://github.com/NOAA-EMC/global-workflow/commit/3b1cf516b2cd3d14b229cade3d39c1468e687d92) corresponds to g-w PR [#3745](https://github.com/NOAA-EMC/global-workflow/pull/3745).  This PR renamed `dev/ci/cases/pr/C96C48_hybatmaerosnowDA.yaml` as `dev/ci/cases/pr/C96C48_hybatmsnowDA.yaml`.  However, GDASApp g-w CI still references `C96C48_hybatmaerosnowDA`.  This g-w CI case no longer exists.  

We should replace `C96C48_hybatmaerosnowDA` with `C96C48_hybatmsnowDA`.  This is more involved than simply changing a string.  GDASApp g-w CI specifies job dependencies for g-w CI cases.  The `C96C48_hybatmsnowDA` case does not need the aerosol job dependencies associated with `C96C48_hybatmaerosnowDA`.  Adding `C96C48_hybatmsnowDA` requires defining the correct job dependencies in GDASApp `test/gw-ci/CMakeLists.txt`. 

The aerosol DA capability is now tested via g-w CI [C96_gcafs_cycled.yaml](https://github.com/NOAA-EMC/global-workflow/blob/develop/dev/ci/cases/pr/C96_gcafs_cycled.yaml).  If we want to GDASApp g-w CI to test aerosol DA, we need to add `C96_gcafs_cycled` to GDASApp g-w CI.

The GDASApp stable-nightly job runs with `WORKFLOW_TESTS="ON"`.  The stable-nightly job will fail during the GDASApp configuration step starting with the 20250607 run.

This issue is opened to document
 - inability to build GDASApp with `WORKFLOW_TESTS="ON"`
 - the need to open a new issue to document addition of a `C96C48_hybatmsnowDA` test to GDASApp g-w CI
 - the need to open a new issue to document addition of a `C96_gcafs_cycled` test to GDASApp g-w CI

--- a/.github/pull_request_template.md
+++ b/.github/pull_request_template.md
@@ -18,7 +18,6 @@ Refs NOAA-EMC/repo#5678
 <!-- Which Global Workflow CI tests are required to adequately test this PR? -->
 - [ ] atm_jjob <!-- JEDI atm single cycle DA !-->
 - [ ] C96C48_ufs_hybatmDA <!-- JEDI atm cycled DA !-->
-- [ ] C96C48_hybatmaerosnowDA  <!-- JEDI aero/snow cycled DA !-->
 - [ ] C48mx500_3DVarAOWCDA <!-- JEDI low-res marine 3DVar cycled DA  !-->
 - [ ] C48mx500_hybAOWCDA <!-- JEDI marine hybrid envar cycled DA !-->
 - [ ] C96C48_hybatmDA <!-- GSI atm cycled DA !-->

--- a/test/gw-ci/CMakeLists.txt
+++ b/test/gw-ci/CMakeLists.txt
@@ -251,7 +251,7 @@ if (WORKFLOW_TESTS)
   # List of tests to run
   option(TEST_GSI "Enable the GFSv17 GSI tests" ON)
   option(TEST_GFS18 "Enable the GFSv18 Atmos JEDI tests" ON)
-  option(TEST_AERO_LAND "Enable the GFSv17 Aero-Land JEDI tests" ON)
+  option(TEST_AERO_LAND "Enable the GFSv17 Aero-Land JEDI tests" OFF)
   option(TEST_MARINE_VAR "Enable the GFSv17 Marine JEDI 3D VAR FGAT tests" ON)
   option(TEST_MARINE_HYB "Enable the GFSv17 Marine JEDI 3d HYB VAR tests" ON)
   option(TEST_GFS17 "Enable the GFSv17 WCDA tests" OFF)

